### DISCUSSION TITLE
How to download latest version from ESP web installer?

### DISCUSSION THREAD

**Discussion by Terrevue** [forkineye/ESPixelStick#793] [General]
Same as title.  It looks like the latest CI build failed but there is a version from 10-21-24 on the web installer (https://espixelstickwebflasher.from-ct.com:5000/).  How can I download the latest version used on the web installer?

**Comment by MartinMueller2003** [comment]
Web installer gets its versions from here: 
https://github.com/MartinMueller2003/ESPixelStick/actions
